screens/PaymentScreen.js

```diff
@@ -19,44 +23,54 @@ export default class PaymentScreen extends React.Component {
     });
   }
 
-  show = mode => {
+  showdatepicker = () => {
     this.setState({
       show: true,
-      mode,
     });
   }
 
-  datepicker = () => {
-    this.show('date');
+  savePayment = () => {
+    const { paymentType, institution, amount, payOut, date } = this.state
+    createPayment(paymentType, institution, amount, payOut, date).then(() => {
+      this.props.navigation.navigate('Home')
+    })
   }
 
   render() {
-    const { show, date, mode } = this.state;
+    const { show, date } = this.state;
     return (
       <View style={{ padding: 30, paddingTop: 50 }}>
-        <View style={{display: 'flex', flexDirection: 'column', height: '70%'}}>
+        <View style={{ display: 'flex', flexDirection: 'column', height: '70%' }}>
```

Comment:
I don't like inline styles :v, but  if you think it's ok..., go ahead, you're the boss your choice  XD